ios/RNWorkersManager.m

```diff
@@ -67,7 +73,20 @@ - (void)setWorker:(RNWorkersInstanceManager *)worker
   // Resolve worker URL using the bundle root and resource, and the bundler port.
 
   BOOL uniquePort = NO;
-  NSURL *workerURL = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:root fallbackResource:resource];
+  NSURL *workerURL;
+
+#if !DEBUG
+  Class CodePush = NSClassFromString(@"CodePush");
+  if (CodePush) {
```

Comment:
all this tested + working?  including after an update?

Reply:
Will ask Mitchell to test now.